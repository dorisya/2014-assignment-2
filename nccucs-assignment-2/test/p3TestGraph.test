# Graph test.

CreateGraph graph1
CreateNode n1 6
CreateNode n2 3
ListNodes graph1
AddNode graph1 n1
AddNode graph1 n2
AddEdge graph1 n1 n2
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2
FindPath graph1 n1 -> n2

CreateNode n3 1
CreateNode n4 2
AddNode graph1 n3
AddNode graph1 n4
AddEdge graph1 n1 n3
AddEdge graph1 n2 n4
AddEdge graph1 n3 n4
FindPath graph1 n1 -> n4