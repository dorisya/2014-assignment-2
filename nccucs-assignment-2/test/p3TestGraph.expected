# Graph test.

created graph graph1
created node n1 with cost 6
created node n2 with cost 3
graph1 contains: 
added node n1 to graph1
added node n2 to graph1
added edge from n1 to n2 in graph1
graph1 contains: n1 n2
the children of n1 in graph1 are: n2
the children of n2 in graph1 are: 
shortest path in graph1: n1 n2

created node n3 with cost 1
created node n4 with cost 2
added node n3 to graph1
added node n4 to graph1
added edge from n1 to n3 in graph1
added edge from n2 to n4 in graph1
added edge from n3 to n4 in graph1
shortest path in graph1: n1 n3 n4